python/turbine_models/tests/stateless_llama_test.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 Nod Labs, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import logging
+import turbine_models.custom_models.stateless_llama as llama
+import unittest
+import os
+import pytest
+
+from typing import Literal
+
+
+def test_export(quantization: Literal["int4", None], precision: Literal["f16", "f32"]):
+    llama.export_transformer_model(
+        hf_model_name="llSourcell/medllama2_7b",
+        hf_auth_token=None,
+        compile_to="vmfb",
+        external_weights="safetensors",
+        # external_weight_file="medllama2_7b_f16_int4.safetensors", Do not export weights because this doesn't get quantized
+        quantization=quantization,
+        precision=precision,
+        device="llvm-cpu",
+        target_triple="host",
+        max_alloc="4294967296",
+    )
+
+    from turbine_models.gen_external_params.gen_external_params import (
+        gen_external_params,
+    )
+
+    gen_external_params(
+        hf_model_name="llSourcell/medllama2_7b",
+        quantization=quantization,
+        weight_path="medllama2_7b_f16_int4.safetensors",
+        hf_auth_token=None,
+        precision=precision,
+    )
+
+    from types import SimpleNamespace
+
+    args = SimpleNamespace()
+    args.hf_model_name = "llSourcell/medllama2_7b"
+    args.hf_auth_token = None
+    args.vmfb_path = "medllama2_7b.vmfb"
+    args.external_weight_file = "medllama2_7b_f16_int4.safetensors"
+    args.run_vmfb = True
+    args.device = "llvm-cpu"
+    args.precision = precision
```

Comment:
are these flags set from the pytest_generate_tests in conftest.py? If so this may be a very long test for a ci as each config will probably be at least 5 min

Reply:
Yup they are. But it only runs one combo unless the `--all` flag is passed to pytest.

We can do regular pytest to run only one config for usual tests, and then pass the all flag for e.g. monthly releases.

Reply:
Cool. Yeah for this CI I think we just want to run one config and eventually have a nightly CI that runs all or at least a larger subset of configs

Reply:
Would like to get the CPU ci in ASAP, and then take some time later to make a more comprehensive CI.